server/src/main/java/model/shooting/ship/ShootingShip.java

```diff
@@ -0,0 +1,29 @@
+package model.shooting.ship;
+
+import model.shooting.board.BoardObserver;
+import model.shooting.board.ShootingBoard;
+import model.Coordinates;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ShootingShip implements BoardObserver {
+    private final List<Coordinates> mastCoordinates;
```

Comment:
Two colllections in one class, to wrapper

Reply:
Left for future consideration https://github.com/totine/OStatki/issues/27